k0459866/assignment02.py

```diff
@@ -0,0 +1,11 @@
+# make a code that calculates something
+# it'll be aabout school days
+
+vars (days = 7)
```

Comment:
I tried to define variables by using strings from other programming
languages, as I've looked into Java, C++, and Javascript, I'm not super
fluent with Python. I edited it recently so that the whole program works,
but the vars was meant to say that that word was a variable equaling the
value after it, but I recently learned that you define variables by putting
the variable and what it's equal to, without the vars string.

-Logan Walsh

On Tue, Aug 30, 2016 at 10:05 PM, Chris Ortman <notifications@github.com>
wrote:

> In k0459866/assignment02.py
> <https://github.com/chrisortman/CIS-121/pull/14#discussion_r76917539>:
>
> > @@ -0,0 +1,11 @@
> > +# make a code that calculates something
> > +# it'll be aabout school days
> > +
> > +vars (days = 7)
>
> What does this line do?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/chrisortman/CIS-121/pull/14/files/91cb361755a965931837ae32e771e461397da75d#r76917539>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AUNySwhaadeQhEJqrmiFWnWzle1VBAGcks5qlO-EgaJpZM4JwhK6>
> .
>



-- 
Logan Walsh